### DISCUSSION TITLE
Will reclaiming package ownership break existing dependencies?

### DISCUSSION THREAD

**Discussion by f1yingbanana** [openupm/openupm#4494] [General]
According to [this guide](https://openupm.com/docs/reclaim-package-ownership.html), we can update an existing package to point to the new repo if we want to take over ownership of a package. However, if we are adding OpenUPM support to a repo, it's likely that the previous versions (releases) of the package won't be present. Wouldn't that break existing dependencies in projects? What's the best way to handle this, if this is indeed an issue?

**Comment by favoyang** [comment]
It should be fine in most cases. The build pipelines will not remove a released version if they discover that the related Git tag is missing.

A forked repo is usually very careful to align the version with the original version, keeping minimal changes and only adding Git tags.

Example:

Forked repo (published on OpenUPM):

- Git tags: 1.0.0 and 1.0.1
- OpenUPM versions: 1.0.0 and 1.0.1

Original repo takes over:

- Git tags: only 1.1.0
- OpenUPM versions: 1.0.0, 1.0.1, and 1.1.0

If you have a special case, we can discuss here.
